src/main/java/com/rusticisoftware/tincan/RemoteLRS.java

```diff
@@ -30,24 +33,29 @@
 import com.rusticisoftware.tincan.json.Mapper;
 import com.rusticisoftware.tincan.json.StringOfJSON;
 import com.rusticisoftware.tincan.v10x.StatementsQuery;
+import com.rusticisoftware.tincan.http.HTTPPart;
+import com.rusticisoftware.tincan.internal.MultipartParser;
```

Comment:
Seems a little thing, but can you move the .http. bits together, and move .internal up between http and json to keep it alphabetical at the same level.